chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -57,7 +58,7 @@ metadata:
     release: {{ .Release.Name }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: {{ .Release.Service }}
-    executor: "{{ .Values.executor }}"
+    executor: "{{ $executors._0 }}"
```

Comment:
We should document this behaviour maybe in `values.yaml` as a comment. To explain to users it includes the first element of the given executor while deploying the scheduler.

Reply:
As you suggested, I included a brief comment in `values.yaml`. I would appreciate any suggestions from anyone for further improvement.